productService/serverless.yml

```diff
@@ -0,0 +1,28 @@
+service: my-store-app-backend
+
+frameworkVersion: "3"
+
+plugins:
+  - serverless-webpack
```

Comment:
NIT: I would recommend to check this [plugin](https://www.npmjs.com/package/serverless-auto-swagger). It looks like [API docs](https://vembfe6b3g.execute-api.eu-central-1.amazonaws.com/swagger)